bin/test-workflow/kubernetes/mysql.template.yml

```diff
@@ -0,0 +1,137 @@
+---
```

Comment:
I'm wondering if we're better off showing the test flow working with a single database. I don't think we gain very much by trying to capture every database. I'm flagging this because there's a lot of these manifests that we have to maintain for the different dbs.

Perhaps a better approach would be to have the app deploy help chart handle the following scenarios
1. Specify your own db credentials and app is deployed to use them
2. Specify a db type (in the helm chart values) and have the helm chart generate the database from a subchart (or manifests bundled into the helm chart). This should also be able to handle Openshift. This would make maintenance a lot simpler.

Reply:
Deleted - only testing with postgres in this round